pkg/api/validation/schema.go

```diff
@@ -219,6 +254,40 @@ func (s *SwaggerSchema) ValidateObject(obj interface{}, fieldName, typeName stri
 	return allErrs
 }
 
+// delegateForApiVersion delegates the validation of an object in a different ApiGroup to another SwaggerSchema.
+// Will panic if obj is in the same api group as this SwaggerSchema.
+func (s *SwaggerSchema) delegateForApiVersion(obj interface{}) error {
```

Comment:
this code path should probably be using runtime.Unstructured because it overlaps with the accessors and work @krousey has been doing there.  Not a hard requirement, but I don't like us propagating accessor code.